packages/sandboxd/spec/jupyter/jupyter.proto

```
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package sandboxd.jupyter.v1;
```

Buf (1.70.0)

[error] Files with package "sandboxd.jupyter.v1" must be within a directory "sandboxd/jupyter/v1" relative to root but were in directory "packages/sandboxd/spec/jupyter".

(PACKAGE_DIRECTORY_MATCH)